charts/inception/values.yaml

```diff
@@ -0,0 +1,495 @@
+podAnnotations: {}
+
+# -- Extra labels for INCEpTION's pod.
+# @section -- INCEpTION deployment settings
+podLabels: {}
+
+# -- User-defined init-container to run before INCEpTION pod, as defined in [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/).
+# @section -- INCEpTION deployment settings
+initContainers: {}
+
+# -- User-defined extra containers to run alongside INCEpTION pod, as defined in [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/pods/#using-pods).
+# @section -- INCEpTION deployment settings
+extraContainers: {}
+
+# -- Extra environment variables for INCEpTION's container.
+# @section -- INCEpTION deployment settings
+extraEnv: {}
+
+# -- Extra volume definition for INCEpTION's pod.
+# @section -- INCEpTION deployment settings
+extraVolumes: {}
+# @section -- INCEpTION deployment settings
+
+# -- Extra volume mounts for INCEpTION's pod.
+# @section -- INCEpTION deployment settings
+extraVolumeMounts: {}
+
+# INCEpTION container image settings.
+image:
+  # -- Image repository.
+  # @section -- INCEpTION deployment settings
+  repository: "ghcr.io/inception-project/inception"
+  # -- Image Pull Policy.
+  # @section -- INCEpTION deployment settings
+  pullPolicy: "IfNotPresent"
+  # -- Image tag.
+  # @section -- INCEpTION deployment settings
+  tag: "33.3"
+
+# -- INCEpTION container image Pull Secret.
+# @section -- INCEpTION deployment settings
+imagePullSecrets: {}
+
+# INCEpTION's pod startup probe.
+startupProbe:
+  # -- Enables the startup probe.
+  # @section -- INCEpTION deployment settings
+  enabled: true
+  # -- Startup probe initial startup delay.
+  # @section -- INCEpTION deployment settings
+  initialDelaySeconds: 30
+  # -- Startup probe check interval, in seconds.
+  # @section -- INCEpTION deployment settings
+  periodSeconds: 10
+  # -- Startup probe's number of accepted failures.
+  # @section -- INCEpTION deployment settings
+  failureThreshold: 5
+
+# INCEpTION's pod liveness probe.
+livenessProbe:
+  # -- Enables the liveness probe.
+  # @section -- INCEpTION deployment settings
+  enabled: true
+  # -- Liveness probe initial startup delay.
+  # @section -- INCEpTION deployment settings
+  initialDelaySeconds: 0
+  # -- Liveness probe check interval, in seconds.
+  # @section -- INCEpTION deployment settings
+  periodSeconds: 5
+  # -- Liveness probe's number of accepted failures.
+  # @section -- INCEpTION deployment settings
+  failureThreshold: 2
+
+# INCEpTION podSecurityContext.
+podSecurityContext:
+  # -- User ID to run as.
+  # @section -- INCEpTION deployment settings
+  runAsUser: 2000
+  # -- Group ID to run as.
+  # @section -- INCEpTION deployment settings
+  runAsGroup: 2000
+  # -- Group ID authorized to mount external PVCs.
+  # @section -- INCEpTION deployment settings
+  fsGroup: 2000
+  # -- Allowing container(s) to run as root.
+  # @section -- INCEpTION deployment settings
+  runAsNonRoot: true
```

Comment:
Is it necessary to set that? I believe INCEpTION should be able to run on Kubernetes as a non-privileged user.